ext/openssl/ossl_ssl.c

```diff
@@ -1322,15 +1322,92 @@ ossl_sslctx_add_certificate(int argc, VALUE *argv, VALUE self)
     return self;
 }
 
+/*
+ * call-seq:
+ *    ctx.add_certificate_chain_file(certs_path, pkey_path) -> self
+ *
+ * Loads chain certificates from _certs_path_ and a private key from
+ * _pkey_path_.
+ *
+ * === Parameters
+ * _certs_path_::
+ *   A path to a chain certificates file. It may be a single certificate.
+ *   An instance of String.
+ * _pkey_path_::
+ *   A path to a private key file. An instance of String.
+ *
+ * === Example
+ *   ctx.add_certificate_chain(rsa_cert_path, rsa_key_path)
```

Comment:
Is this correct? should it be `_file`?

Reply:
Yes. `SSLContext#add_certificate_chain` call `SSL_CTX_use_certificate_chain_file`.
It receives the file path to the certificate chain.
https://www.openssl.org/docs/manmaster/man3/SSL_CTX_use_certificate_chain_file.html

For example, `SSLContext#add_certificate_chain` receives file paths at [test_ssl.rb#196](https://github.com/ruby/openssl/blob/417e8cffa50241c9f0c89a40e4d4d2febd4340e3/test/test_ssl.rb#L196)
Rewriting as below, it raises `TypeError`
```ruby
- ctx.add_certificate_chain_file(certs.path, pkey.path)
+ ctx.add_certificate_chain_file(certs, pkey)
```

Reply:
`rsa_cert_path` variable name was insufficient, so I modified.

Reply:
Oh.. Thank you!! Fixed.